src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -92,9 +93,23 @@ export class AddressForm extends React.Component<
               Country
             </Serif>
             <CountrySelect
-              selected={this.state.address.country}
+              selected={
```

Comment:
I played with the idea of keeping the `AddressForm` agnostic about what it means to lock addresses to the continental US, but couldn't figure out a nice enough way to do it, I'm open to suggestions though. I was considering passing in objects of `disabled` and `description` similar to how we do for errors, so that the consumer could make the form continental US only by setting US as the default, disabling the field, and setting "Ships to continental US only" string as the description. It seemed relatively heavy though.

Reply:
I think logic-wise this is OK, but `disableCountrySelect` makes this read in a confusing way (since it's just just about putting a `disabled={true}` on the country field). Can we rename to something a little more descriptive? `lockCountryToUS`? (this is a hard one)